packages/terra-dev-site/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## Unreleased
 
+## 8.1.0 - (June 7, 2022)
+
 * Changed
-  * updated `terra-application` version in peer dependency.
+  * Support for date-time functions in ie 10.
```

Comment:
terra-dev-site has only peer dependency update changes. : https://github.com/cerner/terra-application/pull/296 . 

date-time support changes are done only for terra-polyfill : https://github.com/cerner/terra-application/pull/297 

So the previous change-log is valid for terra-dev-site : `updated `terra-application` version in peer dependency.`

Reply:
Now I understand, thanks. I'll revert it back to how it see.